### DISCUSSION TITLE
ROSCO tuning file .yaml for NREL 5MW FOWT

### DISCUSSION THREAD

**Discussion by void1981** [NatLabRockies/ROSCO#235] [Q&A]
I am trying to use ROSCO as a reference controller for the NREL 5MW ITI Barge floating offshore wind turbine model, with the tuning file [NREL5MW.yaml](https://github.com/NREL/ROSCO/blob/41f2c0015dbdeac1307ba798d7590c61a40bd11e/Tune_Cases/NREL5MW.yaml) the results are much worse than OpenFAST baseline controller [Discon_ITIBarge.dll](https://github.com/OpenFAST/openfast/releases/download/v3.4.1/Discon_ITIBarge.dll).

Is there any guideline for adjusting this tuning file `NREL5MW.yaml` for the ITI Barge model?
and [ROSCO documentation](https://rosco.readthedocs.io/en/latest/source/toolbox_input.html) is not really helpful here.

**Comment by dzalkind** [comment]
Can you share some time series about the behavior you are expecting?  Do you have any suggestions for the documentation?
